dima/react-movies/src/components/contentContainer/contentContainer.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from "react";
+import "./contentContainer.css";
+interface ContainerProps {
+  children: React.ReactChild[] | React.ReactChild | JSX.Element | JSX.Element[];
```

Comment:
looks like unnecessary long list of types. **children : React.ReactNode** would be enough. 
https://github.com/typescript-cheatsheets/react#useful-react-prop-type-examples

Reply:
fixed